Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -28,6 +28,7 @@
 
 using Google.Apis.Download;
 using Google.Apis.Services;
+using System.Net;

```

Comment:
Using statement should be ordered.
First system
<empty line>
3rd party
<empty line>
Google usings.

In your case, just move it before this line:
https://github.com/google/google-api-dotnet-client/blob/master/Src/GoogleApis/Apis/%5BMedia%5D/Download/MediaDownloader.cs#L20